### ISSUE TITLE
Beowulf's second attack doesn't step forward after being charged

### ISSUE THREAD

**Issue by BroadestOfSwords** [muhopensores/dmc3-inputs-thing#9]
The second attack in Beowulf's combo doesn't step forward while being charged with inertia on. See [here](https://gfycat.com/dimpledwatchfullhasaapso)

Note how Dante 'slides' backwards slightly instead of stepping forward. The move appears to function normally when not charged with inertia enabled.

**Comment by BroadestOfSwords** [comment]
Sometimes something similar will occur when getting up from being knocked down but again not consistently: Dante won't move and the animation for getting up plays with him in a static position. Potentially related?

**Comment by muhopensores** [comment]
Yeah the way we shoehorned inertia into existing player velocity logic is causing that weird behavior, in dmc4 it looks like they had some separate logic for air/ground but here it's unified. In the end i decided to just list all exceptions but it was certain i would miss some things. I'll try to figure out how to avoid that or just give up and add more exceptions. Thanks!

**Comment by BroadestOfSwords** [comment]
You're welcome. It's probably also happening [here](https://twitter.com/BD_Swords/status/1433608916038873121), too.
